Assets/Scripts/Classes/User.cs

```diff
@@ -1,7 +1,19 @@
-using UnityEngine;
-
+//Singleton class
 public class User
 {
+    private static User instance = null;
+    private User(){}
+
+    public static User Instance
+    {
+        get{
+            if(instance == null){
+                instance = new User();
+            }
+            return instance;
+        }
+    }
+
```

Comment:
Here we need a coment, to better understand the class.

Reply:
Done